upgrading.md

```diff
@@ -0,0 +1,36 @@
+Upgrading OpenSearch Python Client
+==================================
+
+### Upgrading to >= 2.2.0
+
+Opensearch-py now includes opensearch-dsl-py features. Opensearch-dsl-py was merged into opensearch-py in the following PR [#287](https://github.com/opensearch-project/opensearch-py/pull/287). 
+
+
+#### Overview of architectural changes:
+
+Opensearch-py functionalities that already exist are not altered. Only the following areas have new additions.
+
+The opensearchpy "helpers" module now provides access to aggs, analysis, document, faceted search, field, function, index, mapping, query, search, update by query, utils, and wrappers. As a result, without importing opensearch-dsl-py, these functionalities can be imported directly from opensearch-py. The "connections" file from DSL is now included in the "connection" module. Additionally, the dsl files "serializer" and "exceptions" are combined with their equivalent opensearchpy files "serializer" and "exceptions".
```

Comment:
* Add a link to the helpers module.
* Add a link to the user guide where users can see how to use the new APIs.
* I think its okay if we don't mention how the files were merged.